proto/options.proto

```diff
@@ -158,6 +158,7 @@ message Options {
     transit_available = 9;
     expansion = 10;
     centroid = 11;
+    status = 12;
```

Comment:
for now we only have the action. in a future PR we should make a status message and fill it with fields that we expect to populate in loki/thor/odin